fe2-android/app/src/main/AndroidManifest.xml

```diff
@@ -16,7 +16,9 @@
     android:roundIcon="@mipmap/ic_launcher_round"
     android:supportsRtl="true"
     android:theme="@style/AppTheme">
-    <activity android:name="com.github.dedis.popstellar.ui.home.HomeActivity">
+    <activity
+      android:name="com.github.dedis.popstellar.ui.home.HomeActivity"
+      android:exported="false">
```

Comment:
With exported="false", the app won't launch and the IDE says "A launchable activity must be exported as of Android 12"
```suggestion
      android:exported="true">
```

Reply:
This is ... weird ... I don't have this issue and from the documentation, and exported activity can be launched from other apps.

I will fix this

Reply:
I was using the emulator with Pixel 2, API 29.
Do we need to update something in android studio ?